test/extended/util/certs/ownership/generate.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/image"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Fprintf(os.Stderr, "error: requires exactly one argument\n")
+		os.Exit(1)
+	}
+	outputDir := os.Args[1]
+
+	image.InitializeImages("")
+	exutil.WithCleanup(func() {})
+
+	// TODO[vrutkovs]: read this from openshift/api
+	files := exutil.FixturePaths("testdata", "tls", "ownership")
+	for _, fileName := range files {
```

Comment:
I think we want to read from the entire rawTLS directory and transform into your struct from library-go: https://github.com/openshift/library-go/blob/master/pkg/certs/cert-inspection/certgraphapi/type_registry.go#L4

with keys based on namespace/name

Reply:
imo its easier to group by owner - respective teams would have to make changes in one file, which is better UX